Tests/Unit/EmogrifierTest.php

```diff
@@ -1090,4 +1090,45 @@ public function emogrifyMergesCssWithMixedUnits()
             $this->subject->emogrify()
         );
     }
+
+    /**
+     * @test
+     */
+    public function emogrifyRemovesDisplayNoneElements()
```

Comment:
Please add the precondition for this behavior to the test name (also for the other test): emogrifyFor...RemovesInvisibleElement